Newsletters: add 187 (2022-02-16)

harding opened the pull request:
- Client/services section @bitschmidty 
- Eclair 2164 @dongcarl 
- Other stuff @harding 

(Sorry, there are probably problems with this PR.  My laptop broke and I haven't yet restored my ability to preview the newsletter locally.)

Xekyo commented:
@harding: Anything we can do to help?

Xekyo commented on _posts/en/newsletters/2022-02-16-newsletter.md:
```diff
@@ -0,0 +1,64 @@
+  input as `OP_TXHASH`.  The new opcode would make serialized fields
+  from a spending transaction available to a [tapscript][topic
+  tapscript].  Scripts could then test the fields directly (e.g. is the
+  transaction version's number greater than `1`?) or hash the data and
```
```suggestion
  transaction's version number greater than `1`?) or hash the data and
  ```

Xekyo reviewed (commented):
I edited the title of the PR to amend the date, but the commit message has still the wrong issue and date.

Xekyo commented:
I built the site locally, everything looks alright, and all links included so far resolve to the correct site.

Xekyo commented:
I updated the commit message to reference the correct issue and date.

bitschmidty reviewed (commented):
Just one suggestion so far

harding commented:
Made suggested edit (thanks @bitschmidty!) and reviewed edits and additions by @Xekyo, @bitschmidty, and @dongcarl (thanks!), all of which look great to me.  Removed releases/RC section (nothing this week) and added topic links.

Thanks everyone, doubly so for the extra help while I was have tech issues!

bitschmidty commented on _posts/en/newsletters/2022-02-16-newsletter.md:
```diff
@@ -57,10 +57,10 @@ wallets and services.*
   trusted auditor, also noting shortcomings and future improvements. Kraken
-  proves onchain address ownership by digitally signing,
-  a merkle tree of Kraken user account balances is created, the auditor verifies
+  proves onchain address ownership by digitally signing
```
@harding Im not sure this is right. I believe Kraken is signing to prove ownership of UTXOs onchain. Not signing a merkle tree

harding replied:
Sorry!  Hopefully fixed in the latest push.

Xekyo replied:
That mention of Kraken's POR made me wonder how they show each user separate UTXOs, and especially how they prove that. It doesn't seem particularly scalable to maintain each user's balance as distinct UTXOs.

Xekyo replied:
Ah nevermind. Rereading that section, I realize now that the user balances on the exchange was included in the total and the notary certifies that the on-chain balance exceeds all deposits. It's much clearer now!

Xekyo reviewed (commented):
Looks good to me in another pass! I can rebuild it locally when I get home later and check the links again.

adamjonas commented on _posts/en/newsletters/2022-02-16-newsletter.md:
```diff
@@ -0,0 +1,103 @@
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Eclair #2164][] improves its handling of feature bits in various contexts.
+  Notably, invoices that requires a mandatory but non-invoice feature would no
```
```suggestion
  Notably, invoices that require a mandatory but non-invoice feature would no
```

glozow commented:
LGTM other than the s/requires/require - thanks!

bitschmidty commented:
Pushed a commit for @adamjonas feedback, ensured local build/checks were successful, squashed and merged.

Thanks @harding @dongcarl @Xekyo @glozow @adamjonas !
